corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -165,19 +165,25 @@ FormplayerFrontend.on('startForm', function (data) {
 
 FormplayerFrontend.on("start", function (options) {
     var user = FormplayerFrontend.request('currentUser'),
+        savedDisplayOptions,
         appId;
     user.username = options.username;
     user.language = options.language;
     user.apps = options.apps;
     user.domain = options.domain;
     user.formplayer_url = options.formplayer_url;
     user.debuggerEnabled = options.debuggerEnabled;
-    user.phoneMode = options.phoneMode;
     user.restoreAs = FormplayerFrontend.request('restoreAsUser', user.domain, user.username);
-    user.displayOptions = {
+
+    savedDisplayOptions = _.pick(
+        Util.getSavedDisplayOptions(),
+        FormplayerFrontend.Constants.ALLOWED_SAVED_OPTIONS
+    );
```

Comment:
essentially we'll load the saved display options from localStorage. the key is namespaced by domain and username